### PR TITLE
Rename display ads vars in the views

### PR DESCRIPTION
**Pull request by lightalloy**
## What type of PR is this? (check all applicable)

- [x] Refactor

## Description
Renamed variables in the views and a couple of views themselves

## Related Tickets & Documents
- Related Issue #19372

### CODE REVIEW

File: app/views/stories/_main_stories_feed.html.erb [hunk 1]
```diff
@@ -10,48 +10,48 @@
...
 
 <% if @featured_story.id && @pinned_article&.id != @featured_story.id %>
-  <% second_display_ad_position -= 1 %>
+  <% second_billboard_position -= 1 %>
   <%= render partial: "articles/single_story", locals: { story: @featured_story, featured: true, pinned: false } %>
 <% end %>
 
 <div id="article-index-podcast-div"></div>
 
 <%# This third-display-ad should be shown between 7th and 8th article. %>
```
Comment (rt4914) [inline app/views/stories/_main_stories_feed.html.erb:32]: @lightalloy Need to fix this comment too.

```suggestion
<%# This third-billboard should be shown between 7th and 8th article. %>
```

File: app/views/stories/_main_stories_feed.html.erb [hunk 2]
```diff
@@ -10,48 +10,48 @@
...
-<% third_display_ad_position = second_display_ad_position + 5 %>
+<% third_billboard_position = second_billboard_position + 5 %>
 
 <div class="substories" id="substories">
   <% if @stories.any? %>
     <% @stories.each_with_index do |story, i| %>
-      <% if !user_signed_in? && i == second_display_ad_position %>
+      <% if !user_signed_in? && i == second_billboard_position %>
         <% @sidebar_ad_second = DisplayAd.for_display(area: "feed_second", user_signed_in: user_signed_in?) %>
```
Comment (rt4914) [inline app/views/stories/_main_stories_feed.html.erb:39]: Optional: Can we rename this to `sidebar_billboard_second ` ?

File: app/views/stories/_main_stories_feed.html.erb [hunk 3]
```diff
@@ -10,48 +10,48 @@
...
         <% if @sidebar_ad_second %>
-          <%= render partial: "shared/display_ad", locals: { display_ad: @sidebar_ad_second, data_context_type: BillboardEvent::CONTEXT_TYPE_HOME } %>
+          <%= render partial: "shared/billboard", locals: { billboard: @sidebar_ad_second, data_context_type: BillboardEvent::CONTEXT_TYPE_HOME } %>
         <% end %>
       <% end %>
 
-      <% if !user_signed_in? && i == third_display_ad_position %>
+      <% if !user_signed_in? && i == third_billboard_position %>
         <% @sidebar_ad_third = DisplayAd.for_display(area: "feed_third", user_signed_in: user_signed_in?) %>
         <% if @sidebar_ad_third %>
```
Comment (rt4914) [inline app/views/stories/_main_stories_feed.html.erb:47]: Ditto as above. 
Have a look at `sidebar_ad_first` too at top.

**Review by rt4914 — APPROVED** [review]
Approved with some minor suggestions.

File: app/views/stories/_main_stories_feed.html.erb (hunk 2 above)
Comment (lightalloy) [reply app/views/stories/_main_stories_feed.html.erb]: Renamed this and other `sidebar_ad_*`s

**Review by Ridhwana — APPROVED** [review]
LGTM 🚀

**Pull request merged**